src/js/background/background.js

```diff
@@ -64,6 +64,38 @@ async function getAliasesFromServer(method = "GET", opts=null) {
   return masks;
 }
 
+// This function is defined as global in the ESLint config _because_ it is created here:
+// eslint-disable-next-line no-redeclare, no-unused-vars
+async function patchMaskInfo(method = "PATCH", id, data, opts=null) {
+
+  const { relayApiSource } = await browser.storage.local.get("relayApiSource");  
+  const updateRelayAddressURL = `${relayApiSource}/relayaddresses/${id}/`;
+  const updateDomainAddressURL = `${relayApiSource}/domainaddresses/${id}/`;
+
+  const csrfCookieValue = await browser.storage.local.get("csrfCookieValue");
+  const headers = new Headers();
+  
+  headers.set("X-CSRFToken", csrfCookieValue);
+  headers.set("Content-Type", "application/json");
+  headers.set("Accept", "application/json");
+  
+  if (opts && opts.auth) {
+    const apiToken = await browser.storage.local.get("apiToken");
+    headers.set("Authorization", `Token ${apiToken.apiToken}`);
+  }
```

Comment:
Wasn't there a function somewhere that did all this?

Reply:
Yes! Replaced with [`createNewHeadersObject()` function](https://github.com/mozilla/fx-private-relay-add-on/blob/main/src/js/background/background.js#L170-L186). Replaced in ﻿﻿80402c57eaa99b1b1617691f015a18f1ed6fda02